doc/ref_model/chapters/chapter02.md

```diff
@@ -1,51 +1,51 @@
 [<< Back](../../ref_model)
-# 2 VNF Requirements & Analysis
+# 2 Workload Requirements & Analysis
 <p align="right"><img src="../figures/bogo_lsf.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
-* [2.1 VNFs Collateral.](#2.1)
-* [2.2 Analysis.](#2.2)
-* [2.3 NFVI Profiles.](#2.3)
+* [2.1 Workloads Collateral](#2.1)
+* [2.2 Analysis](#2.2)
+* [2.3 Cloud Infrastructure Profiles](#2.3)
 
-The NFV Infrastructure (NFVI) is the totality of all hardware and software components which build up the environment in which VNFs/VAs are deployed, managed and executed. It is, therefore, inevitable that different VNFs/VAs would require different capabilities and have different expectations from it.
+The Cloud Infrastructure is the totality of all hardware and software components which build up the environment in which VNFs/CNFs (workloads) are deployed, managed and executed. It is, therefore, inevitable that different workloads would require different capabilities and have different expectations from it.
 
-One of the main targets of the CNTT is to define an agnostic NFVI and removes any dependencies between VNFs/VAs and the deployed Infrastructure (NFVI) and offer NFVI to VNFs/VAs in an abstracted way with defined capabilities and metrics.
+One of the main targets of the CNTT is to define an agnostic cloud infrastructure, to remove any dependencies between workloads and the deployed cloud infrastructure, and offer infrastructure resources to workloads in an abstracted way with defined capabilities and metrics.
 
-This means, operators will be able to host their Telco Workload (VNF) with different traffic types, behaviour and from any vendor on a unified consistent Infrastructure.
+This means, operators will be able to host their Telco Workloads (VNFs/CNFs) with different traffic types, behaviour and from any vendor on a unified consistent Cloud Infrastructure.
```

Comment:
I think we do not need to capitalize.

Reply:
Agree - fixed in https://github.com/cntt-n/CNTT/pull/1328/commits/7c280ea895694e46b20f9b886d4f5f240ec30f90